src/sentry/runner/__init__.py

```diff
@@ -46,19 +46,20 @@ def cli(ctx, config):
 
 # TODO(mattrobenolt): Autodiscover commands?
 map(lambda cmd: cli.add_command(import_string(cmd)), (
+    'sentry.runner.commands.backup.export',
```

Comment:
Ah, I didn't catch that. I just sorted based on the result of `sort` — I can switch it back if you have a strong preference. (I don't really have one, other than this is marginally easier to maintain.)